src/components/application_manager/src/rpc_handler_impl.cc

```diff
@@ -218,9 +218,30 @@ bool RPCHandlerImpl::ConvertMessageToSO(
 
       rpc::ValidationReport report("RPC");
 
+      // Attach RPC version to SmartObject if it does not exist yet.
+      auto app_ptr = app_manager_.application(message.connection_key());
+      utils::SemanticVersion msg_version(2, 0, 0);
+      if (app_ptr &&
+          (output[NsSmartDeviceLink::NsJSONHandler::strings::S_PARAMS]
+               .keyExists(NsSmartDeviceLink::NsJSONHandler::strings::
+                              S_RPC_MSG_VERSION) == false)) {
+        msg_version = app_ptr->msg_version();
+        output[NsSmartDeviceLink::NsJSONHandler::strings::S_PARAMS]
+              [NsSmartDeviceLink::NsJSONHandler::strings::S_RPC_MSG_VERSION] =
+                  msg_version.toString();
```

Comment:
Also, does this need to be set? Is it used anywhere, or was it entirely replaced by the `msg_version` variable?